webapp/channels/src/components/admin_console/license_settings/modals/confirm_license_removal_modal.scss

```diff
@@ -1,4 +1,15 @@
 .ConfirmLicenseRemovalModal {
+    .modal-header {
+        border-bottom: none !important;
```

Comment:
wondering why we have to use !important

Reply:
@M-ZubairAhmed I don't remember, I think I can remove that.